docs/mathematics/sum-of-divisors.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Sum of Divisors
+slug: /mathematics/sum-of-divisors
+---
+
+## Introduction
+
+Sum of divisors refers to the sum of numbers in the range `1 to n` which perfectly divides `n`.
+
+## Explanation 
+
+We will make a sum variable and initialize it `0`. Then we will make a variable `i` and iterate it on the range `[1, sqrt(n)]` and will check whether `n` is divisible by `i` or not. If `n` is divisible by `i` and not equal to `sqrt(n)` we will add `i` and `n/i` to sum as both `i` and `n/i` will be the divisor of `n` because `i*(n/i) = n` and `i` lies on the left side of `sqrt(n)` and `n/i` on the right. If `n` is divisible by `i` and is equal to `sqrt(n)` we will only add `i` to sum as both `i` and `n/i` will be the same. 
+
```

Comment:
Make it more intuitive and readable with normal English text and not jargons. Use the `Algorithm` section to detail about the formula and steps